README.md

```diff
@@ -108,9 +114,18 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://www.sendinblue.com/ - 9000 mails per months for free
 
 ## CDN and Protection
+
   * http://www.cloudflare.com/ - Basic service is free, good for a blog etc
   * http://www.bootstrapcdn.com/ - CDN for bootstrap, bootswatch and font awesome
-  * https://surge.sh - Zero-bullshit, single–command, bring your own source control web publishing CDN.
+  * https://surge.sh/ - Zero-bullshit, single–command, bring your own source control web publishing CDN
+  * http://www.jsdelivr.com/ - CDN for libraries, plugins, fonts, CSS... can submit if one is missing
+  * https://tech.yandex.ru/jslibs/ - CDN for JavaScript in Russia
+  * http://www.asp.net/ajax/cdn - M$ Ajax CDN, jQuery, ASP.net, JSHint, Globalize, Knockout, Bootstrap
+  * http://www.cdnperf.com/ - Find the fastest CDN for your actual site
+
+## DNS
+
+  * https://nsone.net/ - Data Driven DNS, automatic traffic management, 1M free Queries 
```

Comment:
This one probably belongs to IaaS - trying to limit the amount of categorise if possible